.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,44 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Does this work like a wildcard? I can not find anything about this in the action documentation.

Reply:
I don't understand the question

Reply:
The action is defined to be triggered by pull requests, but it's not specified by what pull requests. I guess by just adding the category without any branches it works like:

```
  pull_request:
    branches:
      - *
```

Reply:
yes, I guess it's the same. I used it in other projects and it triggers on every pull request